cnf-certification-test/identifiers/identifiers.go

```diff
@@ -362,6 +362,10 @@ var (
 		Url:     formTestURL(common.LifecycleTestKey, "persistent-volume-reclaim-policy"),
 		Version: versionOne,
 	}
+	TestContainersImageTag = claim.Identifier{
+		Url:     formTestURL(common.LifecycleTestKey, "test-containers-image-tag"),
```

Comment:
I have not seen the prefix "test" often in our suite. I'd use "containers-image-tag"